.ci_scripts/sphinx_markdown_to_docusaurus.py

```diff
@@ -0,0 +1,78 @@
+"""
+Given the Markdown output of a sphinx site, place it in the correct place for Docusaurus.
+"""
+import sys
+from pathlib import Path
+
+from ruamel.yaml import YAML
+
+yaml = YAML(typ="safe")
+yaml.default_flow_style = False
+
+SIDEBAR_ORDERING = [
```

Comment:
If we change the website's high-level structure, does that mean this list will need to change too? If yes, we should add a comment and maybe have some sort of CI check.

Reply:
Good catch! Added a check to raise if the document is not found in this list. Hopefully, if we start changing `sphinx/` stuff, is to bring it over to Docusaurus, so "this should not happen" but better be safe than sorry.